packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -62,13 +63,20 @@ class _SnapshotItemContent extends StatelessWidget {
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
                 Expanded(
-                  child: Markdown(
-                    data: _snapshotDocumentation,
-                    onTapLink: (text, url, title) async =>
-                        await launchUrl(url!),
+                  child: ListView(
+                    children: [
+                      SizedBox(
+                        height: 450,
+                        child: Markdown(
+                          data: _snapshotDocumentation,
+                          onTapLink: (text, url, title) async =>
```

Comment:
Nit: does this need to be `async`? I'm not sure if the call to `onTapLink` is `await`ed.